### PR TITLE
[WIN32SS] Fix the last remaining menu regressions

### PR DESCRIPTION
**Pull request by learn-more**
JIRA issue: [CORE-15863](https://jira.reactos.org/browse/CORE-15863)

### CODE REVIEW

**Comment by JoachimHenze** [comment]
I tested all commits up to and including 4b6b68f. Thumbs up for committing from my side.
Read my in-detail test with videos in https://jira.reactos.org/browse/CORE-15863?focusedCommentId=116713&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-116713

Without any doubt this is the best state we ever had up to now in ros.

File: win32ss/user/ntuser/menu.c [hunk 1]
```diff
@@ -3439,6 +3457,12 @@ static PMENU FASTCALL MENU_ShowSubPopup(PWND WndOwner, PMENU Menu, BOOL SelectFi
       }
   }
 
+  /* Next menu does not need to be shown vertical anymore */
+  if (Menu->fFlags & MNF_POPUP)
+      Flags &= (~TPM_VERTICAL);
+
+
```
Comment (katahiromz) [inline win32ss/user/ntuser/menu.c]: extra empty lines

**Comment by katahiromz** [comment]
Can I commit #1814 before #1812 ?

**Comment by katahiromz** [comment]
`WS_EX_DLGMODALFRAME` and `WS_BORDER` will affect visual. It will be double border.

**Comment by learn-more** [comment]
> 
> 
> `WS_EX_DLGMODALFRAME` and `WS_BORDER` will affect visual. It will be double border.

So the menu code would need to be changed to adapt to this?

**Comment by katahiromz** [comment]
OK. I will wait.

**Pull request merged**